local-cli/runAndroid/runAndroid.js

```diff
@@ -71,10 +71,38 @@ function checkAndroid(args) {
   return fs.existsSync(path.join(args.root, 'android/gradlew'));
 }
 
+function getAdbPath() {
+  return process.env.ANDROID_HOME
+    ? process.env.ANDROID_HOME + '/platform-tools/adb'
+    : 'adb';
+}
+
+// Runs ADB reverse tcp:8081 tcp:8081 to allow loading the jsbundle from the packager
```

Comment:
Minor thing: Runs 'adb reverse tcp:8081 tcp:8081' to allow loading the JS bundle from the packager when using a physical device

- adb is usually spelled in lower case: https://developer.android.com/studio/command-line/adb.html
- JS bundle is the usual spelling
- we should say the purpose of this function is running on physical devices (on emulators it's not needed)